model/holder.go

```diff
@@ -0,0 +1,11 @@
+package model
+
+type Holder struct {
+	PoolAdrress  string `json:"pool_address"`
+	PoolType     string `json:"pool_type"`
+	Address      string `json:"user_address"`
+	ProjectCode  string `json:"project_code"`
+	TokenAddress string `json:"token_address"`
+	TokenType    string `json:"token_type"`
```

Comment:
```suggestion
	TokenType    string `json:"token_type"`
```

Không cần lưu token type

Reply:
Nên bổ sung chain để phân biệt chain